packit_service/worker/monitoring.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+
+from prometheus_client import CollectorRegistry, Counter, push_to_gateway
+
+logger = logging.getLogger(__name__)
+
+
+class Pushgateway:
+    def __init__(self):
+        self.pushgateway_address = (
+            "http://pushgateway" if os.getenv("DEPLOYMENT") in ("stg", "prod") else None
+        )
+        # so that workers don't overwrite their metrics,
+        # the job name corresponds to worker name (e.g. packit-worker-0)
+        self.worker_name = os.getenv("HOSTNAME")
+        self.registry = CollectorRegistry()
+
+        # metrics
+        self.copr_builds_created = Counter(
+            "copr_builds_created_count",
+            "Count of created copr-builds",
```

Comment:
I think it's not required to indicate the type of the metric in the name, as that one is going to be reported in the endpoint.

And `_total` and `_created` seems to be automatically added by Prometheus. (Saying this based on how `github_webhook_calls` shows up in https://prod.packit.dev/metrics .)

```suggestion
        self.copr_builds = Counter(
            "copr_builds",
            "Number of Copr builds created",
```